static/themes/common/base.css

```diff
@@ -320,7 +320,8 @@
     color: var(--brand-default-fg);
 }
 
-.kiwi-userbox .kiwi-userbox-actions .kiwi-userbox-action:hover {
+.kiwi-userbox .kiwi-userbox-actions .kiwi-userbox-action:hover,
```

Comment:
This line can simply be `.kiwi-userbox-action:hover,`

Reply:
This has been made less specific now